npm install fails on node 6.2.0

Running `npm install` on node -v 6.2 seems to fail. With nvm I ran install with -v 4.4.4 and no errors occurred 
```
> kerberos@0.0.17 install /Users/klaas/Code/relax/node_modules/kerberos

> (node-gyp rebuild) || (exit 0)

  CXX(target) Release/obj.target/kerberos/lib/kerberos.o

In file included from ../lib/kerberos.cc:1:

In file included from ../lib/kerberos.h:9:

../../nan/nan.h:590:20: error: no type named 'GCEpilogueCallback' in 'v8::Isolate'

      v8::Isolate::GCEpilogueCallback callback

      ~~~~~~~~~~~~~^

../../nan/nan.h:596:20: error: no type named 'GCEpilogueCallback' in 'v8::Isolate'

      v8::Isolate::GCEpilogueCallback callback) {

      ~~~~~~~~~~~~~^

../../nan/nan.h:601:20: error: no type named 'GCPrologueCallback' in 'v8::Isolate'

      v8::Isolate::GCPrologueCallback callback

      ~~~~~~~~~~~~~^

../../nan/nan.h:607:20: error: no type named 'GCPrologueCallback' in 'v8::Isolate'

      v8::Isolate::GCPrologueCallback callback) {

      ~~~~~~~~~~~~~^

4 errors generated.

make: *** [Release/obj.target/kerberos/lib/kerberos.o] Error 1

gyp ERR! build error 

gyp ERR! stack Error: `make` failed with exit code: 2

gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)

gyp ERR! stack     at emitTwo (events.js:106:13)

gyp ERR! stack     at ChildProcess.emit (events.js:191:7)

gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:204:12)

gyp ERR! System Darwin 15.4.0

gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"

gyp ERR! cwd /Users/klaas/Code/relax/node_modules/kerberos

gyp ERR! node -v v6.2.0

gyp ERR! node-gyp -v v3.3.1

gyp ERR! not ok 

```

--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
     "hoist-non-react-statics": "^1.0.3",
     "invariant": "^2.2.0",
     "js-md5": "^0.3.0",
-    "kerberos": "0.0.17",
+    "kerberos": "^0.0.21",
     "keymaster": "git+https://github.com/relax/keymaster.git",
     "lodash.clone": "^3.0.3",
     "lodash.clonedeep": "^3.0.2",

